### ISSUE TITLE
Cont.dat keeps writing text mode

### ISSUE THREAD

**Issue by FLO-2DKaren** [FLO-2DSoftware/qgis-flo-2d-plugin#866]
@FLO-2DJJ 

Hi JJ,

If you try to set timdep settings, the plugin keeps turning off Display Graphics mode and changing it to text mode.  Can you check that?  There shouldn't be any triggers that relate the time series to the Display Graphics.

**Comment by FLO-2DJJ** [comment]
I'll check it again.

**Comment by FLO-2DJJ** [comment]
@FLO-2DKaren I do not see the problem. I set LGPLOT to Display Graphics and combined it with all options of ITIMTEP and TIMTEP. CONT.DAT is written as expected: the 3rd value of 1st. line of CONT.DAT is 3 for Display Graphics in all cases.
Is there other situation that you are seeing? If so, could you give me the details, please?

**Comment by FLO-2DKaren** [comment]
@FLO-2DJJ 

here are the steps that make it fail on my computer. 

1. Set to Detailed Graphics.
2. Set Graphic Time switch.
3. Set Time Series Switches.
4. Save and close dialog.
5. Reload Dialog and you see that it is reset to Text mode.

**Comment by FLO-2DKaren** [comment]
@FLO-2DJJ 

Let me try again to simplify.

1. Set to Detailed Graphics.
2. Set Update Time Interval to 0.1.
3. Set Timeseries output to any number 0 - 5.
4. Set Output Interval to 0.5.
5. Set Use Time Interval to anything you want.
6. Save and Close.
7. ReOpen dialog box and see that Detailed Graphics is now back to Text Mode.

**Comment by FLO-2DKaren** [comment]
That's what happens on my computer.  Just want to confirm it's happening for you too.

**Comment by FLO-2DJJ** [comment]
Thanks Karen,
I followed your 7 steps. The Detailed Graphics remains the same.
Don't know why you get something else. Would you like to see the issue
together with me?

Juan Jose Rodriguez
FLO-2D Software
www.flo-2d.com
***@***.***

**Comment by FLO-2DKaren** [comment]
OK then.  Let’s eliminate this for now.
